app/controllers/groups_controller.rb

```diff
@@ -294,6 +327,23 @@ def add_members
     end
   end
 
+  def membership_request
+    group = Group.find(params[:id])
+    guardian.ensure_can_edit!(group)
+
+    user = User.find(params[:user_id])
```

Comment:
This is going to raise an error if the `id` is invalid. As a result, the next line isn't going to work.